styles/variables-base.less

```diff
@@ -671,6 +671,7 @@
 @agate-tooltip-label-height: 45px;
 @agate-tooltip-label-font-weight: @agate-button-font-weight;
 @agate-tooltip-padding: 3px 18px;
+@agate-tooltip-marquee-width: 600px;
```

Comment:
Could you check it work for default value?

Tooltip grows unlimitly despite I checking tooltipMarquee.

Reply:
It doesn't apply this width because of this style block:
`&.marquee {
				max-width: var(--agate-tooltip-label-width, @agate-tooltip-marquee-width); // If using a marquee, use the preferred width, and if none is provided, fall back to our default variable.
			}`
I did this to follow the Sandstone structure, but by modifying:
`max-width: @agate-tooltip-marquee-width`, it will only go up to maximum 600px when `tooltipMarquee` is true. Do you think we should leave it like this?

Reply:
Ah, I checked the sandstone status.
hm.. I likes the current status but there is one problem when I set tooltipWidth bigger than 600. 
it looks the tooltipWidth not be applied.  It would be better to return to the previous state.